sql_generators/glean_usage/templates/baseline_clients_daily_v1.query.sql

```diff
@@ -24,22 +24,37 @@ WITH base AS (
     normalized_channel,
     normalized_os,
     normalized_os_version,
-    {% if has_distribution_id %}
-    metrics.string.metrics_distribution_id AS distribution_id,
-    {% else %}
-    CAST(NULL AS STRING) AS distribution_id,
-    {% endif %}
-    {% if app_name == "fenix" %}
-    metrics.string.first_session_install_source AS install_source,
-    {% else %}
-    CAST(NULL AS STRING) AS install_source,
-    {% endif %}
     metadata.geo.subdivision1 AS geo_subdivision,
     {% if has_profile_group_id %}
     metrics.uuid.legacy_telemetry_profile_group_id AS profile_group_id,
     {% else %}
     CAST(NULL AS STRING) AS profile_group_id,
     {% endif %}
+    {% if app_name == "firefox_desktop" %}
+    client_info.windows_build_number as windows_build_number,
```

Comment:
```suggestion
    client_info.windows_build_number AS windows_build_number,
```

I believe our SQL style guide mentions that we should capitalize reserved words:
https://docs.telemetry.mozilla.org/concepts/sql_style#reserved-words